api/compose/api.go

```diff
@@ -37,7 +37,7 @@ type Service interface {
 	// Up executes the equivalent to a `compose up`
 	Up(ctx context.Context, project *types.Project, detach bool) error
 	// Down executes the equivalent to a `compose down`
-	Down(ctx context.Context, projectName string) error
+	Down(ctx context.Context, projectName string, removeOprhans bool) error
```

Comment:
Could this be an object rather than a boolean?

Reply:
I don't get your point here. Would you prefer of a `DownOptions` struct to hold this bool?

Reply:
Yes exactly